--- ./nginxdata/nginx.conf.bak	2026-09-07 22:12:26.438625100 -0700
+++ ./nginxdata/nginx.conf	2026-09-07 22:12:46.890568000 -0700
@@ -4,14 +4,14 @@
     server {
         # Redirect all HTTP to HTTPS
         listen 80;
-        server_name drownedashes.com next.drownedashes.com www.drownedashes.com next.tlnnegm.local www.tlnnegm.local tlnnegm.local;
+        server_name torfit.com;
         return 301 https://$host$request_uri;
     }
     server {
         listen	443 ssl;
-        ssl_certificate /etc/nginx/certs/drownedashes_com_chain.crt;
-        ssl_certificate_key	/etc/nginx/certs/drownedashes_com.key;
-        server_name drownedashes.com next.drownedashes.com www.drownedashes.com next.tlnnegm.local www.tlnnegm.local tlnnegm.local;
+        ssl_certificate /etc/nginx/certs/torfit_com_chain.crt;
+        ssl_certificate_key	/etc/nginx/certs/torfit_com.key;
+        server_name torfit.com;
 	client_max_body_size 2048M;
 	proxy_busy_buffers_size 512k;
 	proxy_buffers 4 512k;
@@ -47,14 +47,14 @@
     server {
         # Redirect all HTTP to HTTPS
         listen 80;
-        server_name ntfy.tlnnegm.local ntfy.drownedashes.com;
+        server_name torfit.com;
         return 301 https://$host$request_uri;
     }
     server {
         listen	443 ssl;
-        ssl_certificate /etc/nginx/certs/drownedashes_com_chain.crt;
-        ssl_certificate_key	/etc/nginx/certs/drownedashes_com.key;
-        server_name ntfy.tlnnegm.local ntfy.drownedashes.com ;
+        ssl_certificate /etc/nginx/certs/torfit_com_chain.crt;
+        ssl_certificate_key	/etc/nginx/certs/torfit_com.key;
+        server_name torfit.com;
 
         location / {
             proxy_headers_hash_max_size 512;
@@ -73,16 +73,16 @@
     server {
         # Redirect all HTTP to HTTPS
         listen 80;
-        server_name n8n.tlnnegm.local n8n.drownedashes.com;
+        server_name torfit.com;
         return 301 https://$host$request_uri;
     }
 
     server {
         listen 443 ssl ;
-        server_name n8n.tlnnegm.local n8n.drownedashes.com;
+        server_name torfit.com;
 
-        ssl_certificate /etc/nginx/certs/drownedashes_com_chain.crt;
-        ssl_certificate_key /etc/nginx/certs/drownedashes_com.key;
+        ssl_certificate /etc/nginx/certs/torfit_com_chain.crt;
+        ssl_certificate_key /etc/nginx/certs/torfit_com.key;
 
         ssl_protocols TLSv1.2 TLSv1.3;
         ssl_prefer_server_ciphers on;
@@ -119,20 +119,20 @@
     server {
         # Redirect all HTTP to HTTPS
         listen 80;
-        server_name ai.tlnnegm.local ai.drownedashes.com ;
+        server_name torfit.com;
         return 301 https://$host$request_uri;
     }
 
     server {
         listen 443 ssl ;
-        ssl_certificate /etc/nginx/certs/drownedashes_com_chain.crt;
-        ssl_certificate_key	/etc/nginx/certs/drownedashes_com.key;
+        ssl_certificate /etc/nginx/certs/torfit_com_chain.crt;
+        ssl_certificate_key	/etc/nginx/certs/torfit_com.key;
         ssl_protocols TLSv1.2 TLSv1.3;
         ssl_prefer_server_ciphers on;
         ssl_session_timeout 5m;
         ssl_session_cache shared:SSL:50m;
         ssl_session_tickets off;
-        server_name ai.tlnnegm.local ai.drownedashes.com ;
+        server_name torfit.com;
 
         location / {
             proxy_headers_hash_max_size 512;
@@ -155,16 +155,16 @@
     server {
         # Redirect all HTTP to HTTPS
         listen 80;
-        server_name jellyfin.tlnnegm.local jellyfin.drownedashes.com;
+        server_name torfit.com;
         return 301 https://$host$request_uri;
     }
 
     server {
         listen 443 ssl ;
-        server_name jellyfin.tlnnegm.local jellyfin.drownedashes.com;
+        server_name torfit.com;
 
-        ssl_certificate /etc/nginx/certs/drownedashes_com_chain.crt;
-        ssl_certificate_key /etc/nginx/certs/drownedashes_com.key;
+        ssl_certificate /etc/nginx/certs/torfit_com_chain.crt;
+        ssl_certificate_key /etc/nginx/certs/torfit_com.key;
 
         ssl_protocols TLSv1.2 TLSv1.3;
         ssl_prefer_server_ciphers on;
@@ -200,16 +200,16 @@
     server {
         # Redirect all HTTP to HTTPS
         listen 80;
-        server_name ntfy.tlnnegm.local ntfy.drownedashes.com;
+        server_name torfit.com;
         return 301 https://$host$request_uri;
     }
 
     server {
         listen 443 ssl ;
-        server_name ntfy.tlnnegm.local ntfy.drownedashes.com;
+        server_name torfit.com;
 
-        ssl_certificate /etc/nginx/certs/drownedashes_com_chain.crt;
-        ssl_certificate_key /etc/nginx/certs/drownedashes_com.key;
+        ssl_certificate /etc/nginx/certs/torfit_com_chain.crt;
+        ssl_certificate_key /etc/nginx/certs/torfit_com.key;
 
         ssl_protocols TLSv1.2 TLSv1.3;
         ssl_prefer_server_ciphers on;
@@ -245,16 +245,16 @@
     server {
         # Redirect all HTTP to HTTPS
         listen 80;
-        server_name adguard.tlnnegm.local adguard.drownedashes.com;
+        server_name torfit.com;
         return 301 https://$host$request_uri;
     }
 
     server {
         listen 443 ssl ;
-        server_name adguard.tlnnegm.local adguard.drownedashes.com;
+        server_name torfit.com;
 
-        ssl_certificate /etc/nginx/certs/drownedashes_com_chain.crt;
-        ssl_certificate_key /etc/nginx/certs/drownedashes_com.key;
+        ssl_certificate /etc/nginx/certs/torfit_com_chain.crt;
+        ssl_certificate_key /etc/nginx/certs/torfit_com.key;
 
         ssl_protocols TLSv1.2 TLSv1.3;
         ssl_prefer_server_ciphers on;
@@ -313,4 +313,4 @@
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     }
     }
-}
+}
\ No newline at end of file
